### PR TITLE
missing derived_from

### PR DESCRIPTION
**Pull request by burrbull**
Parsing part of #81 

cc @Disasm

### CODE REVIEW

**Comment by burrbull** [comment]
bors try

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - [breaking-change] `Defaults` field renamed on `RegisterProperties`
   and added into `Peripheral` and `ClusterInfo`
+- Added `derived_from` into `field` and `ClusterInfo`
```
[the remark this answers is not in the document]
Comment (burrbull) [reply CHANGELOG.md]: Done.

**Comment by Disasm** [comment]
Overall LGTM. Could you add (or modify existing) tests for this field?

**Comment by burrbull** [comment]
I've changed test for `Field`.

File: src/svd/field.rs [hunk 2]
```diff
@@ -193,49 +200,27 @@ mod tests {
...
                     modified_write_values: None,
                     _extensible: (),
                 },
                 "
-            <field>
+            <field derivedFrom=\"other field\">
               <name>MODE</name>
               <bitOffset>24</bitOffset>
               <bitWidth>2</bitWidth>
-              <access>read-write</access>
```
Comment (Disasm) [inline src/svd/field.rs]: Why did you remove this part of the test? Also I'd rather add non-empty `derivedFrom` to the previous test: it contains more features.

Comment (burrbull) [reply src/svd/field.rs]: This test repeat previous almost completely. And we usually don't have other fields when have `derivedFrom`.

Comment (Disasm) [reply src/svd/field.rs]: Makes sense!
